Makefile

```diff
@@ -5,6 +5,9 @@ all: get-deps code-analysis test
 config-install:
 	mkdir -p "$(HOME)/.config/perun"
 	cp defaults/main.yaml "$(HOME)/.config/perun/main.yaml"
+	mkdir -p "$(HOME)/perun"
+	cp defaults/blocked.json "$(HOME)/perun/blocked.json"
```

Comment:
I believe it is still different in comparison to above configuration: `.config/perun/` - I'd love to see something like this:

```bash
 config-install:
 	mkdir -p "$(HOME)/.config/perun/stack-policies/"
 	cp defaults/main.yaml "$(HOME)/.config/perun/main.yaml"
  cp defaults/blocked.json "$(HOME)/.config/perun/stack-policies/blocked.json"
  ...
```